server/helpers_test.go

```diff
@@ -36,3 +38,60 @@ type stubOCSPRespSource struct {
 func (s *stubOCSPRespSource) Fetch(*certchain.RawChain, func() time.Time) (*certchain.OCSPResponse, futureevent.Event, error) {
 	return s.data, futureevent.NeverOccurs(), nil
 }
+
+//------------------------
```

Comment:
50 lines being duplicated is not like 5 lines being repeated for trivial logic, so worth considering consolidation. Given `stubCache` will likely be replaced or rewritten with a new on-memory cache implementation relatively soon, I suggest adding a todo.

Reply:
Added TODO.